docs/advanced/object_literal.md

```diff
@@ -0,0 +1,304 @@
+## Object Literal
+In TypeScript, you can use object literals to create objects for the purpose of function invocation.
+
+### Supported Patterns
+
+```yaml
+name: Advanced Object Literal
+```
+
+#### Semantic: Object Literal
+
+##### Examples
+
+###### Add key
+
+```ts
```

Comment:
If not necessary, do not write `ts` code block, the content below is purely JavaScript code without any TypeScript feature, so please change `ts` to `js`.